src/storage/lock_manager/lock_waiting_queue.rs

```diff
@@ -804,10 +837,25 @@ mod tests {
             encountered_lock_ts: impl Into<TimeStamp>,
             resumable: bool,
         ) -> TestLockWaitEntryHandle {
-            let lock_info_pb = self.make_lock_info_pb(key, encountered_lock_ts);
+            self.mock_lock_wait_with_shared(key, start_ts, encountered_lock_ts, resumable, false)
+        }
+
+        fn mock_lock_wait_with_shared(
```

Comment:
It is confusing to name it `must lock with shared` while having a `shared` parameter.